metaconfig-core/shared/src/main/scala/metaconfig/ConfDecoder.scala

```diff
@@ -60,25 +64,20 @@ object ConfDecoder {
 
   def instanceExpect[T](expect: String)(
       f: PartialFunction[Conf, Configured[T]]
-  )(implicit ev: ClassTag[T]): ConfDecoder[T] =
-    new ConfDecoder[T] {
-      override def read(any: Conf): Configured[T] =
-        f.applyOrElse(
-          any,
-          (x: Conf) => {
-            NotOk(ConfError.typeMismatch(expect, x))
-          }
-        )
-    }
+  ): ConfDecoder[T] = new ConfDecoder[T] {
+    override def read(any: Conf): Configured[T] =
+      f.applyOrElse(any, (x: Conf) => NotOk(ConfError.typeMismatch(expect, x)))
+  }
 
   def constant[T](value: T): ConfDecoder[T] = new ConfDecoder[T] {
-    override def read(conf: Conf): Configured[T] = Configured.ok(value)
```

Comment:
For future PRs, please try to avoid unnecessary diffs like here since it makes reviewing the changes harder.